codesrc/src/main/java/com/onehealth/core/exceptions/NoEventDetailsFoundException.java

```diff
@@ -0,0 +1,5 @@
+package com.onehealth.core.exceptions;
+
+public class NoEventDetailsFoundException extends RuntimeException {
```

Comment:
Lets refactor the type of exceptions. We can only have one DataNotFoundException for any no details found cases

Reply:
ok.